meeting-minutes/team-minutes/nov16-team-minutes.md

```diff
@@ -0,0 +1,11 @@
+# Meeting Minutes - November 16th
+
+## Agenda Items:
+- 「Aleks」Reminder that I will not be able to attend meeting on 23rd
+- 「Aleks」Reminder that meeting on The 30th is now on 23rd

```

Comment:
I think you meant 29th?

Reply:
@aleksssk can you clarify this before I approve?

Reply:
Fixed! thanks for letting me know